browse/config.py

```diff
@@ -210,6 +210,9 @@
 SQLALCHEMY_ECHO = False
 SQLALCHEMY_RECORD_QUERIES = False
 
+# Disable DB queries even if other SQLAlchemy config are defined
+BROWSE_DISABLE_DATABASE = os.environ.get('BROWSE_DISABLE_DATABASE', False)
```

Comment:
My thinking here was to have a clean way to support something like the `no-write` runlevel. This will prevent the query from actually being executed when we know the DB is unavailable.

Reply:
Ah, got it.